pkg/cloud/services/ssm/helpers.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ssm
+
+import "bytes"
+
+func splitBytes(data []byte, maxSize int, iterFunc func([]byte)) {
```

Comment:
Can we dedupe this from both the secrets manager and ssm backends into a single shared library, probably in `pkg/internal/splitbytes` ?